rotkehlchen/tests/api/test_evm_transactions.py

```diff
@@ -89,3 +89,36 @@ def test_query_transactions(rotkehlchen_api_server: 'APIServer'):
         for entry in result['entries']:
             assert entry['entry']['timestamp'] <= last_ts
             last_ts = entry['entry']['timestamp']
+
+
+@pytest.mark.parametrize('optimism_accounts', [[
+    '0xc37b40ABdB939635068d3c5f13E7faF686F03B65',
+]])
+@pytest.mark.parametrize('should_mock_price_queries', [True])
+@pytest.mark.parametrize('default_mock_price_value', [FVal(1.5)])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+def test_query_transactions_single_chain(rotkehlchen_api_server: 'APIServer'):
```

Comment:
This test is not needed. I mentioned it in discord in the past many times but another way to reduce our test time is to do unit tests when required. This is bringing up an entire api server just to test the following:

```
accounts = [x.address for x in query_accounts if x.chain_id == self.evm_inquirer.chain_id] # noqa: E501
```

Also makes a shit-ton of remote calls, which need to be mocked (for new tests be mocking -- do not add todos)

---

Instead just turn this into a unit test:

 So just create an `EthereumTransactions` object, pass it an `EvmTransactionsFilterQuery` with both ethereum and optimism transactions and mock `single_address_query_transactions` to do nothing, but instead read the addresses passed to it. Assert the optimism addresses are not there